src/registry/routes/helpers/get-components-history.ts

```diff
@@ -0,0 +1,32 @@
+import { ComponentsDetails } from '../../../types';
+import dateStringified from '../../../utils/date-stringify';
+
+interface ComponentHistory {
+  name: string;
+  publishDate: number;
+  version: string;
+}
+
+export default function getComponentsHistory(history: ComponentsDetails) {
+  const result: ComponentHistory[] = [];
+
+  for (const [name, versions] of Object.entries(history.components)) {
```

Comment:
why is the order in array switched when compared with original?

Reply:
Because lodash .each/.forEach puts the value first and then the key
https://lodash.com/docs/4.17.15#forEach
But thats not the case with Object.entries